src/kani-compiler/cbmc/src/goto_program/stmt.rs

```diff
@@ -129,6 +129,13 @@ impl Stmt {
             _ => None,
         }
     }
+
+    pub fn get_stmts(&self) -> Option<&Vec<Stmt>> {
```

Comment:
This is part of an earlier version where the intrinsics where encoded in a block statement, so I needed to retrieve statements within in order to prepend the finite checks. This version does not use it, but I figured it could be useful for something else. We can remove it otherwise.

Reply:
If you do keep it, can you please add a comment?

Reply:
Added a comment.